components/eventing-controller/log/log.go

```diff
@@ -0,0 +1,33 @@
+package log
```

Comment:
As calling the main type and all instances logger possibly this package should be called `logger` too.